amundsen_application/static/js/components/common/SearchBar/InlineSearchResults/SearchItemList/SearchItem/index.tsx

```diff
@@ -64,6 +66,7 @@ export class SearchItem extends React.Component<SearchItemProps, {}> {
           <div className="title-2 search-item-info">
             <div className="search-term">{`${searchTerm}\u00a0`}</div>
             <div className="search-item-text">{listItemText}</div>
+            { showBetaFlag && <Flag text="beta" labelStyle="default"/> }
```

Comment:
We could opt to check the `resourceType === ResourceType.dashboard` to show the `Flag`.